### PR TITLE
#161041727 Setup Heroku pipeline

### PR DESCRIPTION
**Pull request by otseobande**
**What does this PR do?**

This PR setups the Heroku pipeline for the application

**Description of Task to be completed?**

- add app.json to enable review apps

**How should this be manually tested?**

- Login to Heroku with the application credentials
- Confirm that the RideMyWay Pipeline is correct

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

- [#161041727](https://www.pivotaltracker.com/story/show/161041727)

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ride-my-way",
+  "scripts": {},
+  "env": {},
+  "formation": {},
+  "addons": [],
+  "buildpacks": []
+}
```
Comment (Annmary12) [inline app.json:8]: leave an extra line

Comment (otseobande) [reply app.json:8]: Lol okay

Comment (otseobande) [reply app.json:8]: There's an extra line, Github removes it when displaying the file. If there wasn't you would see a red arrow under need the file.

Comment (Annmary12) [reply app.json:8]: Okay, Got it

Comment (Annmary12) [reply app.json:8]: Please you need to rebase from develop

**Review by Annmary12 — APPROVED** [review]

**Pull request merged**